aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md

```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
 
 Now you need to get your service bus connection string. In the Azure portal, select the service bus namespace that you created and click the Access Key icon.
 
-![An image of a plus sign labeled 'Create', an image of a key labeled 'Access Key', and an image of a trash can labeled 'Delete'.](scaleout-with-windows-azure-service-bus/_static/image7.png)
+![Graphic of a plus sign labeled Create and a key labeled Access Key and a trash can labeled Delete.](scaleout-with-windows-azure-service-bus/_static/image7.png)
```

Comment:
This is a screenshot, so let's say, "Screenshot that shows a plus sign labeled Create, a key labeled Access Key, and a trash can labeled Delete."